<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Project</class>
 <widget class="QMainWindow" name="Project">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1016</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Project</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: qradialgradient(spread:pad, cx:0.5, cy:0.5, radius:0.5, fx:0.5, fy:0.5, stop:0 rgba(226, 162, 0, 255), stop:1 rgba(255, 255, 255, 255));</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="StartingMenu">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>10</y>
      <width>341</width>
      <height>80</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(134, 94, 255);</string>
    </property>
    <property name="title">
     <string>GroupBox</string>
    </property>
    <widget class="QPushButton" name="StartingMenuButton">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>30</y>
       <width>41</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>ok</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="ManagerNumber">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>30</y>
       <width>141</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(0, 255, 255);</string>
     </property>
     <property name="placeholderText">
      <string>Enter the number here...</string>
     </property>
    </widget>
    <widget class="QLabel" name="StartingMenuLable">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>111</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Number of managers:</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="SignUpBox">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>100</y>
      <width>891</width>
      <height>521</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(243, 255, 116);</string>
    </property>
    <property name="title">
     <string>Sign up box</string>
    </property>
    <widget class="QLabel" name="EmailLable">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>20</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(174, 255, 165);</string>
     </property>
     <property name="text">
      <string>Email:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="EmaillineEdit">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>20</y>
       <width>211</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(0, 255, 255);</string>
     </property>
     <property name="placeholderText">
      <string>Enter your Email here....</string>
     </property>
    </widget>
    <widget class="QPushButton" name="EmailConfirm">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>20</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>Next</string>
     </property>
    </widget>
    <widget class="QPushButton" name="EmailHelp">
     <property name="geometry">
      <rect>
       <x>650</x>
       <y>20</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Help</string>
     </property>
    </widget>
    <widget class="QLabel" name="PasswordLable">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>70</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(174, 255, 165);</string>
     </property>
     <property name="text">
      <string>password:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="PasswordLineEdit">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>70</y>
       <width>211</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(0, 255, 255);</string>
     </property>
     <property name="placeholderText">
      <string>Eneter your password here....</string>
     </property>
    </widget>
    <widget class="QPushButton" name="PasswordHelp">
     <property name="geometry">
      <rect>
       <x>650</x>
       <y>70</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Help</string>
     </property>
    </widget>
    <widget class="QPushButton" name="PasswordConfirm">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>70</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Next</string>
     </property>
    </widget>
    <widget class="QLabel" name="UsernameLabel">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>120</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(174, 255, 165);</string>
     </property>
     <property name="text">
      <string>username:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="UsernameLineEdit">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>120</y>
       <width>211</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(0, 255, 255);</string>
     </property>
     <property name="placeholderText">
      <string>Enter your username here....</string>
     </property>
    </widget>
    <widget class="QPushButton" name="UsernamePushButton">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>120</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Next</string>
     </property>
    </widget>
    <widget class="QPushButton" name="UsernameHelpPushButton">
     <property name="geometry">
      <rect>
       <x>650</x>
       <y>120</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Help</string>
     </property>
    </widget>
    <widget class="QPushButton" name="PasswordSwButton">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>70</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="BudgetLable">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>170</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(174, 255, 165);</string>
     </property>
     <property name="text">
      <string>budget:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="BudgetLineEdit">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>170</y>
       <width>211</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(0, 255, 255);
</string>
     </property>
     <property name="placeholderText">
      <string>Enter your budget here....</string>
     </property>
    </widget>
    <widget class="QPushButton" name="BudgetPushButton">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>170</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Next</string>
     </property>
    </widget>
    <widget class="QPushButton" name="BudgetHelpPushButton">
     <property name="geometry">
      <rect>
       <x>650</x>
       <y>170</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Help</string>
     </property>
    </widget>
    <widget class="QLabel" name="PhoneNumberLabel">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>220</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(174, 255, 165);
</string>
     </property>
     <property name="text">
      <string>Phone number:</string>
     </property>
    </widget>
    <widget class="QComboBox" name="PhoneNumberComboBox">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>220</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 112, 76);</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="PhoneNumberLineEdit">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>220</y>
       <width>161</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(0, 255, 255);</string>
     </property>
     <property name="placeholderText">
      <string>Enter your phone number here....</string>
     </property>
    </widget>
    <widget class="QPushButton" name="PhoneNumberHelpPushButton">
     <property name="geometry">
      <rect>
       <x>760</x>
       <y>220</y>
       <width>75</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Help</string>
     </property>
    </widget>
    <widget class="QLabel" name="AreaCodeLabel">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>220</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(251, 123, 255);
</string>
     </property>
     <property name="text">
      <string>0098</string>
     </property>
    </widget>
    <widget class="QPushButton" name="FinalConfirmPushButton">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>350</y>
       <width>111</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>Confirm</string>
     </property>
    </widget>
    <widget class="QPushButton" name="PhoneNumberushButton">
     <property name="geometry">
      <rect>
       <x>650</x>
       <y>220</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Next</string>
     </property>
    </widget>
    <widget class="QLabel" name="CaptchaLabel">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>260</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(174, 255, 165);</string>
     </property>
     <property name="text">
      <string>Code:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="CaptchaGenerateLineEdit">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>260</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(255, 178, 102, 255), stop:0.55 rgba(235, 148, 61, 255), stop:0.98 rgba(0, 0, 0, 255), stop:1 rgba(0, 0, 0, 0));</string>
     </property>
    </widget>
    <widget class="QPushButton" name="CaptchaGeneratePushButton">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>260</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Retry</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="CaptchaLineEdit">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>260</y>
       <width>161</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(0, 255, 255);</string>
     </property>
     <property name="placeholderText">
      <string>Eneter the code here....</string>
     </property>
    </widget>
    <widget class="QPushButton" name="CaptchaConfirmPushButton">
     <property name="geometry">
      <rect>
       <x>650</x>
       <y>260</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Next</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1016</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>FinalConfirmPushButton</sender>
   <signal>clicked()</signal>
   <receiver>BudgetPushButton</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>448</x>
     <y>379</y>
    </hint>
    <hint type="destinationlabel">
     <x>451</x>
     <y>245</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>FinalConfirmPushButton</sender>
   <signal>clicked()</signal>
   <receiver>UsernamePushButton</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>405</x>
     <y>381</y>
    </hint>
    <hint type="destinationlabel">
     <x>393</x>
     <y>211</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>FinalConfirmPushButton</sender>
   <signal>clicked()</signal>
   <receiver>PasswordConfirm</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>422</x>
     <y>381</y>
    </hint>
    <hint type="destinationlabel">
     <x>443</x>
     <y>184</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>FinalConfirmPushButton</sender>
   <signal>clicked()</signal>
   <receiver>EmailConfirm</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>412</x>
     <y>381</y>
    </hint>
    <hint type="destinationlabel">
     <x>418</x>
     <y>155</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>PhoneNumberushButton</sender>
   <signal>clicked()</signal>
   <receiver>CaptchaGeneratePushButton</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>516</x>
     <y>275</y>
    </hint>
    <hint type="destinationlabel">
     <x>344</x>
     <y>304</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
